src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Intake extends SubsystemBase {
+    private TalonSRX motor = new TalonSRX(Ports.Intake.MOTOR);
+    private Solenoid solenoidR = new Solenoid(Ports.Intake.SOLENOID_RIGHT);
+    private Solenoid solenoidL = new Solenoid(Ports.Intake.SOLENOID_LEFT);
+    private UnitModel unitModel = new UnitModel(Constants.Intake.TICK_PER_METER);
+    private State position = State.CLOSE;
+
+    /**
+     * sets the motor inverted
+     */
+    public Intake() {
+        motor.setInverted(Ports.Intake.IS_INVERTED);
+    }
+
+    /**
+     * @return intake's motor velocity
+     */
+    public double getVelocity(){
+        return unitModel.toVelocity(motor.getSelectedSensorVelocity());
+    }
+
+    /**
+     * @return the state of the solenoids
+     */
+    public State getState(){
+        return position;
+    }
+
+    /**
+     * sets the output of intake's motor by present output (%)
+     */
+    public void setPower(double power){
+        motor.set(ControlMode.PercentOutput,power);

```

Comment:
```suggestion
        motor.set(ControlMode.PercentOutput, power);

```

Reply:
and in general, just use Ctrl+Alt+L in intellij, to make the code looks better

Reply:
I recommend committing through IntelliJ IDEA. Then you can organize your code automatically before each commit and you don't need to worry about it.